licence-checker/README.md

```diff
@@ -19,3 +19,27 @@ suffix. This is an artefact of how the checker searches for the licence.
 The checker is configured using a hjson file, which contains the exact licence
 header, and a list of file patterns to exclude from checking the licence for,
 which is used to exclude vendored and other externally sourced files.
+Additionally, the licence checker can be configured to use regex matching using
+the `match_regex` key set to true.
```

Comment:
Nit: Probably quote "true" here (to help people who might not know our slightly odd hjson style guide).